include/aspect/utilities.h

```diff
@@ -95,7 +95,24 @@ namespace aspect
      */
     template <int dim>
     Point<dim>
-    cartesian_coordinates(const std_cxx11::array<double,dim> &scoord);
+    spherical_to_cartesian_coordinates(const std_cxx11::array<double,dim> &scoord);
+
+    /**
+     * Returns ellispoidal coordinates of a cartesian point. The returned array
```

Comment:
Here and everywhere else, we should capitalize "Cartesian" (because it is based on the latin name of the French mathematician Descartes, so it's a proper name).